.circleci/generate_config_yml.py

```diff
@@ -6,23 +6,24 @@
 """
 
 import os
-import sys
 import shutil
-from collections import namedtuple
+import sys
+from collections import OrderedDict, namedtuple
 
-import cimodel.data.pytorch_build_definitions as pytorch_build_definitions
-import cimodel.data.windows_build_definitions as windows_build_definitions
 import cimodel.data.binary_build_definitions as binary_build_definitions
 import cimodel.data.caffe2_build_definitions as caffe2_build_definitions
-import cimodel.data.simple.macos_definitions
-import cimodel.data.simple.mobile_definitions
+import cimodel.data.pytorch_build_definitions as pytorch_build_definitions
```

Comment:
I guess vscode automatically did the formatting on this. In general we should enforce this anyways.